sway-fmt-v2/src/fmt.rs

```diff
@@ -271,4 +271,39 @@ pub const TEST1: u16 = 10;"#;
             Formatter::format(&mut formatter, Arc::from(sway_code_to_format), None).unwrap();
         assert_eq!(correct_sway_code, formatted_sway_code)
     }
+    #[test]
+    fn test_ty_formatting() {
+        let sway_code_to_format = r#"contract;
+
+enum test_ty {
```

Comment:
I realise this is just a test, but the convention for Rust and Sway types is to use capital camel case for type names, e.g. `TestTy` rather than `test_ty`. Folks sometimes refer to tests and examples upstream as demonstrative of conventions, so we should try to follow them wherever we can.

Reply:
Ohh yeah, of course. I must have missed this one and now it triggers me pretty badly 😄